source/modest/finder/pseudo_class.c

```diff
@@ -77,6 +77,62 @@ bool modest_finder_selector_sub_type_pseudo_class_function_has(modest_finder_t*
     return false;
 }
 
+// concat str1 and str2
+char *concat_string(const char *str1, const char *str2)
```

Comment:
move to `core` module, append to [mycore/utils.c](https://github.com/lexborisov/Modest/blob/master/source/mycore/utils.c) and rename to `mycore_str_cat`

Reply:
ok